testerum-backend/testerum-runner/testerum-runner-cmdline/src/main/kotlin/com/testerum/runner_cmdline/events/execution_listeners/junit/JUnitExecutionListener.kt

```diff
@@ -0,0 +1,84 @@
+package com.testerum.runner_cmdline.events.execution_listeners.junit
+
+import com.testerum.runner.events.execution_listener.BaseExecutionListener
+import com.testerum.runner.events.model.FeatureEndEvent
+import com.testerum.runner.events.model.FeatureStartEvent
+import com.testerum.runner.events.model.ParametrizedTestEndEvent
+import com.testerum.runner.events.model.ParametrizedTestStartEvent
+import com.testerum.runner.events.model.RunnerEvent
+import com.testerum.runner.events.model.ScenarioEndEvent
+import com.testerum.runner.events.model.ScenarioStartEvent
+import com.testerum.runner.events.model.StepEndEvent
+import com.testerum.runner.events.model.StepStartEvent
+import com.testerum.runner.events.model.SuiteEndEvent
+import com.testerum.runner.events.model.SuiteStartEvent
+import com.testerum.runner.events.model.TestEndEvent
+import com.testerum.runner.events.model.TestStartEvent
+import com.testerum.runner.events.model.TextLogEvent
+import java.util.concurrent.LinkedBlockingDeque
+import javax.annotation.concurrent.NotThreadSafe
+
+@NotThreadSafe
+class JUnitExecutionListener : BaseExecutionListener() {
+
+    val eventQueue: LinkedBlockingDeque<RunnerEvent> = LinkedBlockingDeque<RunnerEvent>()
```

Comment:
no need to specify the type of the property in this case, since using the constructor makes it clear